<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>740</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>740</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ÊñáÊú¨ËΩ¨ËØ≠Èü≥</string>
  </property>
  <widget class="QPlainTextEdit" name="plainTextEditContent">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>701</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxVoice">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>389</width>
     <height>63</height>
    </rect>
   </property>
   <property name="title">
    <string>Èü≥Ëâ≤</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="radioButtonXiaoxiao">
      <property name="text">
       <string>ÊôìÊôì(Â•≥)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButtonXiaoyi">
      <property name="text">
       <string>ÊôìÊÑè(Â•≥)</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButtonYunjian">
      <property name="text">
       <string>‰∫ëÈó¥</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButtonYunxi">
      <property name="text">
       <string>‰∫ëÂ∏å</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButtonYunxia">
      <property name="text">
       <string>‰∫ëÈúû</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButtonYunyang">
      <property name="text">
       <string>‰∫ëÈò≥</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButtonPlay">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>‚ñ∂Ô∏è Êí≠Êîæ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>510</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>‚ùå ÂÖ≥Èó≠</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>510</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>‚èπÔ∏è ÂÅúÊ≠¢</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSave">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>510</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>üíæ ‰øùÂ≠ò</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxOtherVoice">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>360</y>
     <width>291</width>
     <height>65</height>
    </rect>
   </property>
   <property name="title">
    <string>ÂÖ∂‰ªñÈü≥Ëâ≤</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QComboBox" name="comboBoxLanguage"/>
    </item>
    <item>
     <widget class="QComboBox" name="comboBoxVoiceName"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBoxGPTSoVITS">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>441</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>GPT-SoVITS</string>
   </property>
   <widget class="QSplitter" name="splitterGPTSoVITS">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>421</width>
      <height>23</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QCheckBox" name="checkBoxUseGPTSoVITS">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>ÂêØÁî®</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelRefAudio">
     <property name="text">
      <string>ÂèÇËÄÉÈü≥È¢ë(3-10s):</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditRefAudio"/>
    <widget class="QPushButton" name="pushButtonSelectRefAudio">
     <property name="text">
      <string>ÈÄâÊã©Êñá‰ª∂...</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
